src/types/insights.ts

```diff
@@ -10,11 +10,21 @@ export type InsightsClientPayload = {
   positions?: number[];
 };
 
-export type InsightsClient = (
+export type InsightsSendEvent = (
   method: InsightsClientMethod,
   payload: InsightsClientPayload
 ) => void;
 
+export type InsightsOnUserTokenChangeMethod = 'onUserTokenChange';
+
+export type InsightsOnUserTokenChange = (
+  method: InsightsOnUserTokenChangeMethod,
```

Comment:
```suggestion
export type InsightsOnUserTokenChange = (
  method: 'onUserTokenChange',
```

why is the method name exported separately?

Reply:
Actually no reason. I'll put it directly into the function type def.